client/my-sites/people/people-list-item/index.jsx

```diff
@@ -53,21 +62,40 @@ class PeopleListItem extends React.PureComponent {
 		return type === 'invite' ? inviteLink : editLink;
 	};
 
+	onResend = event => {
+		const { requestingResend, resendSuccess, siteId, inviteKey } = this.props;
+
+		event.preventDefault();
```

Comment:
Why do we need this? Since it is a button `onClick` handler, there isn't any default handling that we need to prevent, is there?

Reply:
It's a little misleading - the card that the resend button is a child of has an href, which sends the user to the invite details screen if anything inside of the card is clicked (including the resend). I can add a comment explaining, but I couldn't think of a less hacky way to do it.

Reply:
Ah, gotcha. Wouldn’t `stopPropagation` be the proper way to prevent this then?

Reply:
Old habits - thanks for the suggestion, I'll fix it up.

Reply:
Actually, it looks like the parent card becomes an anchor tag when an href is specified, so we need to `preventDefault` to prevent the browser navigation behavior _and_ `stopPropagation` to prevent the `onClick` analytics bump and window scroll.